static/app/views/explore/logs/logFieldsTree.spec.tsx

```diff
@@ -64,18 +75,35 @@ describe('logFieldsTree', () => {
             organization,
             location: LocationFixture(),
           }}
+          tableResultLogRow={{
+            [OurLogKnownFieldKey.PROJECT_ID]: 123,
+            [OurLogKnownFieldKey.ORGANIZATION_ID]: 1,
+            [OurLogKnownFieldKey.BODY]: 'test log body',
+            [OurLogKnownFieldKey.SEVERITY_NUMBER]: 456,
+            [OurLogKnownFieldKey.SEVERITY_TEXT]: 'error',
+            [OurLogKnownFieldKey.TIMESTAMP]: '2025-04-03T15:50:10+00:00',
+          }}
```

Comment:
Bonus points if you type this with `https://github.com/getsentry/sentry/blob/master/static/app/views/explore/logs/constants.tsx#L19` so it doesn't go out of sync by accident